src/SymfonyCorp/Connect/Api/Api.php

```diff
@@ -31,15 +33,15 @@ class Api
 {
     const ENDPOINT = 'https://connect.symfony.com/api';
 
-    private $browser;
+    private $httpClient;
     private $parser;
     private $logger;
     private $endpoint;
     private $accessToken;
 
-    public function __construct($endpoint = null, Browser $browser = null, ParserInterface $parser = null, LoggerInterface $logger = null)
+    public function __construct($endpoint = null, HttpClientInterface $httpClient = null, ParserInterface $parser = null, LoggerInterface $logger = null)
```

Comment:
the type-hint should be replaced by a docblock annotation and passing a `Browser` should trigger a deprecation (and we should then act as if `null` was passed)